src/javascripts/components/editCrew/editCrew.js

```diff
@@ -0,0 +1,34 @@
+import crewData from '../../helpers/data/crewData';
+
+import utils from '../../helpers/utils';
+
+const showCrewForm = (crewId) => {
+  crewData.getCrewById(crewId)
+    .then((response) => {
+      const crew = response.data;
+
+      const domString = `
+        <h2> Edit Crew Member</h2>
+        <form class="edit-crew" id=${crewId}> 
```

Comment:
you'll want to take this `edit-crew` class off here, unless you want to run into the same issues that @MarkyAaronYoung found 🙃

Reply:
done